tests/types/helpers.py

```diff
@@ -3,20 +3,21 @@
 
 import fakeredis.aioredis
 
+from async_rediscache.session import RedisSession
+
 
 class BaseRedisObjectTests(unittest.IsolatedAsyncioTestCase):
     """Base class for Redis data type test classes."""
 
     async def asyncSetUp(self):
-        """Patch the RedisSession to pass in a fresh fakeredis pool for each test."""
+        """Patch the RedisSession to pass in a fresh fakeredis client for each test."""
         self.patcher = unittest.mock.patch("async_rediscache.types.base.RedisSession")
-        self.mock_session = self.patcher.start()
+        self.mock_session: RedisSession = self.patcher.start()
         self.mock_session.get_current_session.return_value = self.mock_session
-        self.mock_session.pool = await fakeredis.aioredis.create_redis_pool()
+        self.mock_session.client = fakeredis.aioredis.FakeRedis()  # noqa
```

Comment:
Can you specify which linter error this ignores?

Reply:
The problem is there isn't an error code for this, as it's not a linting error, it's a static analysis error. Client is a method, so replacing it with a variable is invalid. I will add this in a comment though.

Reply:
Which statical analysis tool is that?

Reply:
It was my IDE. I'd think mypy would raise a similar error, though I haven't tried.

Reply:
Nope, just my IDE that raises it. I don't really know if we should keep it or remove it. If I run an inspection on the entire project, this is the only warning.

Reply:
I think it would be beneficial to narrow it down somehow, does `# type: ignore` work? At least, then, it will still check linting.

Reply:
It does not, no. Pycharm only parses `noqa`s